### ISSUE TITLE
This tool doens't solve "Ubuntu Bash for Windows 10" dark blue color problem?

### ISSUE THREAD

**Issue by Qws** [microsoft/terminal#39]
I've changed color theme with the colortool released by Microsoft, but it didn't affected Ubuntu Bash for Windows 10.

My Windows build number: Microsoft Windows [Version 10.0.16299.98]

I'm using Ubuntu for Windows 10, to connect with my VPS through SSH, this is how it looks, and I would like to make blue a lot lighter...

![afbeelding](https://user-images.githubusercontent.com/3944421/33982951-b465eef2-e0b2-11e7-8c64-371cc2f4558e.png)

**Comment by tpmullan** [comment]
It can solve that problem, but for some reason ubuntu terminals record the color settings in the shortcut.  So it will not use the registry defaults.  You can copy the color values from a theme you like, but that takes a lot of time.  If you use this tool in a newer development build you might be able to run the windows executable from the Linux shell.  But that does not work on the stable version.

**Comment by zadjii-msft** [comment]
@tpmullan It's not just Linux terminals that store the settings in the shortcut, it's all console applications. [This blog post](https://blogs.msdn.microsoft.com/commandline/2017/06/20/understanding-windows-console-host-settings/) describes the process a little bit.

I'm working on a solution to be able to use the colortool within WSL right now, so it should be something fixed for the next release (after Fall Creator's Update)

**Comment by stevenengler** [comment]
A while ago I posted a comment on the blog post [here](https://blogs.msdn.microsoft.com/commandline/2017/08/11/introducing-the-windows-console-colortool/#comment-35685) for a workaround (make sure to look at the parent comment as well).

Here's the general workaround:

Go to `%AppData%\Microsoft\Windows\Start Menu\Programs`, right-click the `Bash on Ubuntu on Windows` shortcut and select `Properties`. Change the target to `cmd.exe` and "run" the shortcut. In the command prompt window, change the colorscheme using the tool and save it. Change the shortcut target back to `C:\Windows\System32\bash.exe ~`, and you should have the correct colors when running bash from the start menu.

**Comment by rivy** [comment]
@zadjii-msft , 

You might want to look at [`concfg`](https://github.com/lukesampson/concfg/blob/master/libexec/import.ps1), specifically the ["clean" command](https://github.com/lukesampson/concfg/blob/c17dd9c9a640dfb0c5f7b73aa5bfc7c8ae970833/libexec/clean.ps1). It resets console program and .lnk specific overrides, so that they use the default settings.

The `concfg` program has been a solution in this problem domain for quite some time, and, additionally, handles almost all console property settings.

**Comment by nikeee** [comment]
I was using a shortcut on my taskbar for Bash. After I changed the theme using the colortool and cmd, the theme stayed the same in Bash.
However, unpinning the Bash from the taskbar and readding it solved the it for me (I used the Windows Search and Richt-Click -> Pin To Taskbar).

Also, you can start a CMD and start bash from there.

**Comment by jemiller0** [comment]
Has any progress been made on this? I have the same problem. I wan't to change the colors in my Debian Linux console window. I was going to do what @stevenengler mentioned above, but, don't know where to fine the shortcut to it. I don't see it in the Start menu as he describes.

**Comment by zadjii-msft** [comment]
Have you tried `color tool -x`? The -x flag is used to emit colors in a Linux compatible way.

________________________________

**Comment by jemiller0** [comment]
Is that a Linux command? I just tried running it from Debian, but, it says command not found. Is there a package I need to install?

I just want to make it so that the colors are what they are if you do a fresh install of Windows 10 with the latest ISO that is on MSDN (the February 2019 version). I just did this yesterday on a new computer and the colors look good. I want to adjust the colors on another computer that has all the latest updates, but, was installed when Windows 10 first came out. I would be happy to fix it using the approach that @stevenengler mentioned. However, I don't know where the link to the command window is located, or if something changed with regard to how WSL windows work in more recent versions of Windows.

**Comment by zadjii-msft** [comment]
My apologies, try with colortool.exe after installing colortool somewhere on your %path%. If you want to run windows binaries from wsl, you need to include the .exe

________________________________

**Comment by jemiller0** [comment]
Perfect! Thanks. That did it. I had to build the app from the source as the binary I had, I think was an older one that didn't have the -x option.

**Comment by jemiller0** [comment]
On an unrelated note. One thing I wish Microsoft would do is create a text based editor for Linux, or make it multi-platform that is similar to the old DOS CUI editor. If I remember correctly, I think it was called DOSEDIT or something along those likes. I.e. create an editor that is easy to use that a menu at the top. I bet a lot of people would use it if you did. Vi is pretty terrible and Emacs has completely different key mappings from what Windows uses. It would be great if there were something that had all the same key mappings and worked like normal Windows editors do, but, was text based so that you could use it on Linux without a GUI.

**Comment by miniksa** [comment]
This looks answered. Closing.
